### DISCUSSION TITLE
Thank you! Ngrok server is great, but...

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1865] [Q&A]
Thank you Ben for adding Ngrok server, is so smooth!!

But for everything be perfect I don't know if you can fix this:

This error shows every time I send pictures to IMG2IMG tab, the prompt goes there but the image don't. So I always need to download and send picture there manually.

![image](https://user-images.githubusercontent.com/121369033/227784207-b44e6d74-faa2-4bd4-8529-97f0ba8b6c23.png)

There's also a Lora problem at the beggning, but is already a post opened about that.

Thank you anyway for always improve the Colab.

**Comment by AiXplorer777** [comment]
How do you use the Ngrok server? Rookie here

**Reply by gilroff** [reply]
create an ngrok account.
look for Your Authtoken.
copy and paste in the notebook.
thats all.

**Reply by AiXplorer777** [reply]
Works beautifully!
Still learning the ropes :)
Thanks!

**Reply by Wallnuty** [reply]
> Works beautifully! Still learning the ropes :) Thanks!

Why would you want to use ngrok? I dont know anything about it,

**Reply by AiXplorer777** [reply]
It seems to increase the reliability of the Gradio interface
